src/index.js

```diff
@@ -20,7 +21,10 @@ app.use(express.static(path.join(__dirname, '../public')));
 
 app.use(bodyParser.json());
 app.use(cookieParser()); //Parse the cookie data (User ID).
-	
+app.use(session({
+    secret: 's3Kr3T',
```

Comment:
This secrete should be in a `.env` file and imported from `../src/config/constants.js`. Kindly check how the constant.js file is used across the project.

Reply:
Sure, I'll fix that